vsc_pdf_template_editor/.flutter-plugins

```diff
@@ -1,2 +1,2 @@
 # This is a generated file; do not edit or check into version control.
-printing=/home/dsyrstad/.pub-cache/hosted/pub.dartlang.org/printing-5.9.2/
+printing=/Users/georgiizubkov/Development/flutter/.pub-cache/hosted/pub.dartlang.org/printing-5.9.2/
```

Comment:
Since these files are ignored now in `vsc_pdf_template_editor/.gitignore`, lets delete them. Same for the top-level `.idea/` files that have been committed. I know they are probably prior to your work, but let's clean them up now.

Also, let's create a top-level .gitignore containing:
```
# IntelliJ related
*.iml
*.ipr
*.iws
.idea/

# The .vscode folder contains launch configuration and tasks you configure in
# VS Code which you may wish to be included in version control, so this line
# is commented out by default.
#.vscode/

# Flutter/Dart/Pub related
**/doc/api/
**/ios/Flutter/.last_build_id
.dart_tool/
.flutter-plugins
.flutter-plugins-dependencies
.packages
.pub-cache/
.pub/
build/
```